intranet/static/themes/afd2016/memes.js

```diff
@@ -22,16 +25,18 @@ $(function() {
         var btn = $(this);
         var form = $(this).parent().parent();
         var endpoint = form.attr("data-get-memes");
-        $.get(endpoint, {}, function(resp) {
-            var res = JSON.parse(resp);
+
+        $.getJSON(endpoint, {}, function(res) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2769576168/issues/source?bid=3212528&fileBranchId=3355681#l29)